appcues/src/main/java/com/appcues/trait/appcues/TooltipPathExt.kt

```diff
@@ -317,18 +320,18 @@ fun calculateTooltipWidth(base: Float, length: Float, radius: Float, roundedBase
     //
     // (3) multiply by two to account for the two sides of the pointer
     //     and add the existing width to the end result
-    return (tan(angle) * radius * roundedBase) + base
+    return (tan(angle) * radius * 2) + base
 }
 
-data class CornerPoint(
+internal data class CornerPoint(
```

Comment:
nice catch on internals, we can generate the dokka docs perhaps on main to make sure nothing unexpected got into public api before release

Reply:
nice todo before 2.0